package/src/hooks/useClipboard/useClipboard.ts

```
@@ -0,0 +1,35 @@
+    if (typeof text !== 'string') {
+      throw new Error('copy function only accepts strings');
+    }
+
+    setIsCopied(false);
+    try {
+      const textToCopy = text || initialText;
+      await navigator.clipboard.writeText(textToCopy);
+      setCopiedText(textToCopy);
+      setIsCopied(true);
+    } catch (err) {
+      throw err;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)